loopy/schedule/tools.py

```diff
@@ -810,6 +810,7 @@ def _order_loop_nests(
 
     from pytools.graph import compute_topological_order as toposort
 
+    emptyset: frozenset[InameStr] = frozenset()
```

Comment:
```suggestion
```

Reply:
Thanks! Surprised that the linter didn't yell. I just reenabled the check.

Reply:
Thanks!